src/map.cpp

```diff
@@ -130,12 +130,11 @@ void Map::setTile(uint16_t x, uint16_t y, uint8_t z, Tile* newTile)
 			items->clear();
 		}
 
-		Item* ground = newTile->getGround();
+		auto ground = newTile->getGround();
```

Comment:
Could this be `const auto`
```suggestion
		const auto ground = newTile->getGround();
```